runtime/onert/backend/acl_cl/KernelGenerator.cc

```diff
@@ -1482,26 +1482,19 @@ void KernelGenerator::visit(const ir::operation::Unpack &node)
   axis = acl_common::ToARMComputeAxis(input_rank, axis, frontend_layout, backend_layout).value();
 
   // Disable applied dim_correction
-  for (const auto &output_index : output_indexes)
+  if (input_tensor->num_dimensions() != input_tensor->info()->num_dimensions())
```

Comment:
I have a question. I just assume that one is number of dim from one IR and another is number of dim from acl. Is my assumption correct?

It is hard to understand that both returns number of dimensions but their values are different... (Not the scope of this PR but we may need some enhancement later, I guess...)

Reply:
input_tensor->num_dimensions() -> Wrapper tensor `backend::ITensor`
input_tensor->info()->num_dimensions() -> Tensor of backend `arm_compute::ITensor`

> I have a question. I just assume that one is number of dim from one IR and another is number of dim from acl. Is my assumption correct?

Yes, It is correct.

> It is hard to understand that both returns number of dimensions but their values are different... (Not the scope of this PR but we may need some enhancement later, I guess...)

Okay, I will change it to getting rank from ir.

Reply:
Thanks!